apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -80,23 +80,15 @@ export class GetNotificationsFeed {
         read: command.query.read,
       },
       { limit: COUNT_LIMIT }
-      /*
-       * todo NV-2161 in version 0.16
-       *  update option as below,
-       *  update below:  hasMore = feed.length < totalCount
-       *  remove totalCount
-       * { skip: command.page * command.limit, limit: command.limit + 1 }
-       */
     );
 
-    const hasMore = this.getHasMore(command.page, command.limit, feed, totalCount);
-
     return {
       data: feed || [],
-      totalCount: totalCount || 0,
-      hasMore,
+      hasMore: feed.length < totalCount,
       pageSize: command.limit,
       page: command.page,
+      skip: command.page * command.limit,
+      limit: command.limit + 1,
```

Comment:
Those two should be in the `getCount` query options. and not in the return value.

Regarding the totalCount removal, I could love to do quick sync :)